docker/dockersettings.py

```diff
@@ -96,3 +96,9 @@ def get_partitioned_database_config(use_partitioned_db):
 ELASTICSEARCH_PORT = 9200
 
 SHARED_DRIVE_ROOT = '/sharedfiles'
+
+S3_BLOB_DB_SETTINGS = {
+    "url": "http://riakcs:9880",
```

Comment:
That should be correct. Docker compose uses [docker networking](https://docs.docker.com/engine/userguide/networking/dockernetworks/) to link containers. It should add entries to `/etc/hosts` using the names specified in the compose file links section.